app/src/main/res/values/component_colors.xml

```diff
@@ -2,17 +2,21 @@
 <resources>
   <color name="component_color_shared_edit_text_cursor_color">@color/color_palette_cursor_color</color>
   <color name="component_color_shared_activity_toolbar_color">@color/color_palette_toolbar_color</color>
+  <color name="component_color_shared_activity_toolbar_shadow_color">@color/color_palette_toolbar_shadow_color</color>
   <color name="component_color_shared_activity_toolbar_text_color">@color/color_palette_toolbar_text_color</color>
   <color name="component_color_shared_screen_primary_background_color">@color/color_palette_primary_background_color</color>
   <color name="component_color_shared_screen_secondary_background_color">@color/color_palette_secondary_background_color</color>
   <color name="component_color_shared_screen_tertiary_background_color">@color/color_palette_tertiary_background_color</color>
   <color name="component_color_shared_content_container_background_color">@color/color_palette_content_container_background_color</color>
   <color name="component_color_shared_topic_card_item_background_color">@color/color_palette_topic_card_item_background_color</color>
+  <color name="component_color_shared_topic_activity_background_color">@color/color_palette_topic_activity_background_color</color>
```

Comment:
It cannot be name as ....topic_activity...` because you are using it in `story_fragment` file too. Rename it.

Reply:
Done.